tests/test_teams.py

```diff
@@ -152,8 +158,9 @@ async def test_extract_records(
             "id": 1296269,
             "name": "Hello-World",
             "node_id": "MDEwOlJlcG9zaXRvcnkxMjk2MjY5",
-            "permission": "admin",
             "url": "https://HOSTNAME/repos/octocat/Hello-World",
+            "role_name": "read",
```

Comment:
note that I am making this "role_name" now in order to be consistent with the GitHub REST api.  It is up to the end user to change it if they want.